app/importers/RTMediaMigration.php

```diff
@@ -378,11 +378,15 @@ function test() {
 		} else {
 			?>
 			<div class="error">
-				<p> <?php esc_html_e( 'Please Backup your ', 'buddypress-media' ); ?>
-					<strong><?php esc_html_e( 'DATABASE', 'buddypress-media' ); ?></strong>
-					<?php esc_html_e( ' and ', 'buddypress-media' ); ?>
-					<strong><?php esc_html_e( 'UPLOAD', 'buddypress-media' ); ?></strong>
-					<?php esc_html_e( ' folder before Migration.', 'buddypress-media' ); ?>
+				<p> 
+					<?php 
+			/* translators: %1$s gets replaced by '<strong>', and %2$s by '</strong>' */
+			echo sprintf(
+				__('Please Backup your %1$sDATABASE%2$s and %1$sUPLOAD%2$s folder before Migration.', 'buddypress-media' )
```

Comment:
Sorry about hte Syntax error. Regarding the WP Coding Standars, I already tried to follow them (i.e. https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#opening-and-closing-php-tags) but probably missed something.